Config file not loaded with version 26.14.2

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

After my update to version 26.14.2, my configuration file is not loaded anymore and instead the default configuration is used in bash.
My .bashrc file uses: eval "$(oh-my-posh init bash --config ~/.config/mytheme.omp.json)" and the ~/.config/mytheme.omp.json exists and is not empty

### Theme

{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "palette": {
    "black": "#262B44",
    "blue": "#4B95E9",
    "green": "#59C9A5",
    "orange": "#F07623",
    "red": "#D81E5B",
    "white": "#E0DEF4",
    "yellow": "#F3AE35"
  },
  "secondary_prompt": {
    "template": "<p:yellow,transparent>\ue0b6</><,p:yellow> > </><p:yellow,transparent>\ue0b0</> ",
    "foreground": "p:black",
    "background": "transparent"
  },
  "transient_prompt": {
    "template": "<p:yellow,transparent>\ue0b6</><,p:yellow> {{ .Folder }} </><p:yellow,transparent>\ue0b0</> ",
    "foreground": "p:black",
    "background": "transparent"
  },
  "console_title_template": "{{ .PWD }}",
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "style": "diamond",
          "leading_diamond": "\u256d\u2500\ue0b6",
          "template": " {{ if .WSL }}WSL at {{ end }}{{.Icon}} ",
          "foreground": "p:black",
          "background": "p:yellow",
          "type": "os"
        },
        {
          "style": "diamond",
          "trailing_diamond": "\ue0b0",
          "template": " {{ if .SSHSession }}\ueba9 {{ end }}{{ .UserName }} ",
          "foreground": "p:black",
          "background": "p:yellow",
          "type": "session"
        },
        {
          "style": "powerline",
          "foreground": "#111111",
          "powerline_symbol": "\ue0b0",
          "background": "#FFD664",
          "type": "upgrade",
	  "properties": {
           "cache_duration": "24h0m0s"
          }
        },
        {
          "properties": {
            "mapped_locations": {
              "/home/pascal/dev": "\ue795 ",
              "/home/pascal/tools/wireshark": "\udb85\ude74 ",
              "/mnt/data/dev": "\ue795 ",
              "/mnt/data/tools/wireshark": "\udb85\ude74 "
            },
            "style": "agnoster_full"
          },
          "style": "powerline",
          "template": " \uea83 {{ path .Path .Location }} ",
          "foreground": "p:white",
          "powerline_symbol": "\ue0b0",
          "background": "p:orange",
          "type": "path"
        },
        {
          "properties": {
            "branch_max_length": 25,
            "fetch_status": true,
            "fetch_upstream_icon": true
          },
          "style": "powerline",
          "template": " {{ if .UpstreamURL }}{{ url .UpstreamIcon .UpstreamURL }} {{ end }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }} ",
          "foreground": "p:black",
          "powerline_symbol": "\ue0b0",
          "background": "p:green",
          "type": "git",
          "foreground_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}p:black{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:white{{ end }}",
            "{{ if gt .Ahead 0 }}p:white{{ end }}"
          ],
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:red{{ end }}",
            "{{ if gt .Ahead 0 }}#49416D{{ end }}",
            "{{ if gt .Behind 0 }}#7A306C{{ end }}"
          ]
        },
        {
          "properties": {
            "fetch_status": true
          },
          "style": "powerline",
          "template": " \ue0a0{{.Branch}} r{{.BaseRev}}{{ if .Working.Changed }} \uf044 {{.Working.String}}{{ end }} ",
          "foreground": "p:black",
          "powerline_symbol": "\ue0b0",
          "background": "p:green",
          "type": "svn",
          "foreground_templates": [
            "{{ if or (gt .Working.Modified 0) (gt .Working.Deleted 0) (gt .Working.Added 0) (gt .Working.Moved 0) }}p:black{{ end }}",
            "{{ if .Working.HasConflicts }}}p:white{{ end }}"
          ],
          "background_templates": [
            "{{ if or (gt .Working.Modified 0) (gt .Working.Deleted 0) (gt .Working.Added 0) (gt .Working.Moved 0) }}p:yellow{{ end }}",
            "{{ if .Working.HasConflicts }}p:red{{ end }}"
          ]
        },
        {
          "style": "powerline",
          "template": " \uf0e7 ",
          "foreground": "p:white",
          "powerline_symbol": "\ue0b0",
          "background": "p:yellow",
          "type": "root"
        },
        {
          "properties": {
            "always_enabled": true
          },
          "style": "diamond",
          "leading_diamond": "<transparent,background>\ue0b0</>",
          "trailing_diamond": "\ue0b4",
          "template": " {{ if gt .Code 0 }}\uf00d {{ reason .Code }}{{ else }}\uf00c{{ end }} ",
          "foreground": "p:white",
          "background": "p:blue",
          "type": "status",
          "background_templates": [
            "{{ if gt .Code 0 }}p:red{{ end }}"
          ]
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "right",
      "segments": [
        {
          "properties": {
            "always_enabled": false,
            "style": "austin",
            "threshold": 500
          },
          "style": "plain",
          "template": " <#fefefe>\ueba2</> {{ .FormattedMs }} ",
          "background": "transparent",
          "type": "executiontime"
        },
        {
          "properties": {
            "charging_icon": "\ue234 ",
            "discharging_icon": "\ue231 ",
            "not_charging_icon": "\ue22f "
          },
          "style": "plain",
          "template": " {{ if not .Error }}{{ .Icon }}{{ .Percentage }}{{ end }}\uf295 ",
          "background": "transparent",
          "type": "battery",
          "foreground_templates": [
            "{{if eq \"Charging\" .State.String}}#40c4ff{{end}}",
            "{{if eq \"Discharging\" .State.String}}#ff5722{{end}}",
            "{{if eq \"Full\" .State.String}}#4caf50{{end}}"
          ]
        },
        {
          "properties": {
            "precision": 2
          },
          "style": "plain",
          "template": " \ue266 {{ round .PhysicalPercentUsed .Precision }}% ",
          "background": "transparent",
          "type": "sysinfo"
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "style": "plain",
          "template": "\u2570\u2500",
          "foreground": "p:yellow",
          "type": "text"
        },
        {
          "properties": {
            "always_enabled": true
          },
          "style": "plain",
          "template": "\ue285\ueab6",
          "foreground": "#e0f8ff",
          "type": "status",
          "foreground_templates": [
            "{{ if gt .Code 0 }}p:red{{ end }}"
          ]
        }
      ],
      "newline": true
    }
  ],
  "tooltips": [
    {
      "properties": {
        "display_default": true
      },
      "style": "diamond",
      "leading_diamond": "\ue0b0",
      "trailing_diamond": "\ue0b4",
      "template": " \ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} ",
      "foreground": "p:white",
      "background": "p:orange",
      "type": "aws",
      "tips": [
        "aws"
      ]
    },
    {
      "properties": {
        "display_default": true
      },
      "style": "diamond",
      "leading_diamond": "\ue0b0",
      "trailing_diamond": "\ue0b4",
      "template": " \uebd8 {{ .Name }} ",
      "foreground": "p:white",
      "background": "p:blue",
      "type": "az",
      "tips": [
        "az"
      ]
    }
  ],
  "version": 3,
  "final_space": true,
  "upgrade_notice": true
}


### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
Version: 26.14.2

Shell: bash (5.2.37(1)-release)

Prompt:

 pascal   ltetools                                                                                                                                                                                                 in bash at 09:37:30

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   1 ms
Path(true)                                -   0 ms
Git(false)                                -   0 ms
Root(false)                               -   0 ms
Status(true)                              -   0 ms
Node(false)                               -   2 ms
Go(false)                                 -   1 ms
Python(false)                             -   2 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 7.920713ms

Cache path: /home/pascal/.cache/oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[DEBUG] 09:37:30.764 debug.go:35 → logging enabled, raw mode: true
[DEBUG] 09:37:30.764 load.go:Load:36 → no config file specified, using default
[TRACE] 09:37:30.764 load.go:Load() - 29.585µs
[DEBUG] 09:37:30.764 home.go:12 → /home/pascal
[TRACE] 09:37:30.764 path.go:Path() - 23.916µs
[DEBUG] 09:37:30.764 file.go:Init:26 → loading cache file: /home/pascal/.cache/oh-my-posh/omp.cache
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: INITVERSIONBASH
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: environment_platform
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: is_wsl
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: upgrade_check
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: upgrade_segment
[TRACE] 09:37:30.764 file.go:Init(/home/pascal/.cache/oh-my-posh/omp.cache) - 96.065µs
[DEBUG] 09:37:30.764 home.go:12 → /home/pascal
[TRACE] 09:37:30.764 path.go:Path() - 17.079µs
[DEBUG] 09:37:30.764 file.go:Init:26 → loading cache file: /home/pascal/.cache/oh-my-posh/omp.cache.e5435e1e-b122-4ab2-b6c8-84179eb0f96c
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: prompt_count_cache
[DEBUG] 09:37:30.764 file.go:Init:47 → loading cache key: template_cache
[TRACE] 09:37:30.764 file.go:Init(/home/pascal/.cache/oh-my-posh/omp.cache.e5435e1e-b122-4ab2-b6c8-84179eb0f96c) - 63.647µs
[DEBUG] 09:37:30.764 file.go:Get:80 → getting cache key: prompt_count_cache with value: 1
[TRACE] 09:37:30.764 terminal.go:setPromptCount() - 4.9µs
[TRACE] 09:37:30.764 terminal.go:GOOS() - 57ns
[DEBUG] 09:37:30.764 terminal.go:setPwd:115 → /mnt/data/dev/ltetools
[TRACE] 09:37:30.764 terminal.go:setPwd() - 17.142µs
[TRACE] 09:37:30.764 terminal.go:Init() - 252.459µs
[TRACE] 09:37:30.764 terminal.go:Shell() - 146ns
[TRACE] 09:37:30.764 terminal.go:Flags() - 51ns
[DEBUG] 09:37:30.764 file.go:Get:80 → getting cache key: template_cache with value: {"SegmentsCache":{"Path":{"Segment":{"Text":"  \u003cLINK\u003efile:/mnt/data/dev/ltetools\u003cTEXT\u003eltetools\u003c/TEXT\u003e\u003c/LINK\u003e ","Index":1},"Location":"/mnt/data/dev/ltetools","Path":"ltetools","Folders":[{"Name":"mnt","Path":"/mnt","Display":false},{"Name":"data","Path":"/mnt/data","Display":false},{"Name":"dev","Path":"/mnt/data/dev","Display":false},{"Name":"ltetools","Path":"/mnt/data/dev/ltetools","Display":false}],"StackCount":0,"Writable":true,"RootDir":false},"Session":{"Segment":{"Text":" pascal ","Index":0},"SSHSession":false},"Status":{"Segment":{"Text":"  ","Index":2},"String":"0","Meaning":"0","Error":false}},"Segments":{},"Var":{},"PWD":"/mnt/data/dev/ltetools","Folder":"ltetools","PSWD":"","UserName":"pascal","HostName":"pascal-levallois","ShellVersion":"5.2.37(1)-release","Shell":"bash","AbsolutePWD":"/mnt/data/dev/ltetools","OS":"fedora","Version":"26.14.2","PromptCount":1,"SHLVL":1,"Jobs":0,"Code":0,"WSL":false,"Root":false}
[TRACE] 09:37:30.764 cache.go:restoreCache() - 85.281µs
[DEBUG] 09:37:30.764 writer.go:Init:106 → terminal program: Unknown
[DEBUG] 09:37:30.764 writer.go:Init:107 → terminal shell: shell
[DEBUG] 09:37:30.764 terminal.go:Getenv:77 → NO DATA
[TRACE] 09:37:30.764 terminal.go:Getenv(OMP_CACHE_DISABLED) - 3.765µs
[DEBUG] 09:37:30.764 file.go:Get:75 → cache key not found: accent_color
[TRACE] 09:37:30.764 colors.go:SetAccentColor() - 4.016µs
[TRACE] 09:37:30.764 terminal.go:Shell() - 69ns
[DEBUG] 09:37:30.764 terminal.go:Getenv:77 → 5.2.37(1)-release
[TRACE] 09:37:30.764 terminal.go:Getenv(POSH_SHELL_VERSION) - 2.966µs
[DEBUG] 09:37:30.764 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 09:37:30.764 text.go:patchTemplate:125 → {{ .Shell }} in {{ .Folder }}
[TRACE] 09:37:30.764 text.go:Render({{ .Shell }} in {{ .Folder }}) - 202.046µs
[TRACE] 09:37:30.764 terminal.go:Flags() - 60ns
[DEBUG] 09:37:30.764 terminal.go:Getenv:77 → NO DATA
[TRACE] 09:37:30.764 terminal.go:Getenv(POSH_CURSOR_LINE) - 3.03µs
[DEBUG] 09:37:30.764 terminal.go:Getenv:77 → NO DATA
[TRACE] 09:37:30.764 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 2.178µs
[TRACE] 09:37:30.764 terminal.go:Flags() - 30ns
[TRACE] 09:37:30.764 terminal.go:Flags() - 30ns
[TRACE] 09:37:30.764 terminal.go:Flags() - 81ns
[DEBUG] 09:37:30.764 home.go:12 → /home/pascal
[TRACE] 09:37:30.764 terminal.go:GOOS() - 69ns
[DEBUG] 09:37:30.764 segment.go:Execute:113 → segment: Status
[DEBUG] 09:37:30.764 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 09:37:30.764 terminal.go:Flags() - 177ns
[DEBUG] 09:37:30.764 segment.go:isToggled:234 → no toggles found
[TRACE] 09:37:30.764 terminal.go:StatusCodes() - 108ns
[DEBUG] 09:37:30.764 home.go:12 → /home/pascal
[TRACE] 09:37:30.764 terminal.go:GOOS() - 113ns
[DEBUG] 09:37:30.764 properties.go:GetString:28 → status_template: {{ .Code }}
[DEBUG] 09:37:30.764 segment.go:Execute:113 → segment: Path
[DEBUG] 09:37:30.764 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.764 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.764 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 09:37:30.764 terminal.go:GOOS() - 79ns
[TRACE] 09:37:30.764 separator.go:Separator() - 67ns
[TRACE] 09:37:30.764 terminal.go:Shell() - 149ns
[TRACE] 09:37:30.764 terminal.go:Shell() - 67ns
[TRACE] 09:37:30.764 terminal.go:Flags() - 143ns
[TRACE] 09:37:30.764 terminal.go:GOOS() - 50ns
[DEBUG] 09:37:30.765 text.go:patchTemplate:125 → {{ .Code }}
[TRACE] 09:37:30.765 terminal.go:GOOS() - 125ns
[DEBUG] 09:37:30.765 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 09:37:30.765 properties.go:GetString:28 → windows_registry_icon: 
[TRACE] 09:37:30.765 text.go:Render({{ .Code }}) - 108.804µs
[DEBUG] 09:37:30.765 home.go:12 → /home/pascal
[DEBUG] 09:37:30.765 segment.go:Execute:113 → segment: Session
[DEBUG] 09:37:30.765 properties.go:GetBool:22 → always_enabled: true
[TRACE] 09:37:30.765 separator.go:Separator() - 86ns
[DEBUG] 09:37:30.765 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.765 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.765 terminal.go:Getenv:77 → NO DATA
[TRACE] 09:37:30.765 terminal.go:Getenv(SSH_CONNECTION) - 9.686µs
[TRACE] 09:37:30.765 terminal.go:Flags() - 169ns
[DEBUG] 09:37:30.765 terminal.go:Getenv:77 → NO DATA
[TRACE] 09:37:30.765 terminal.go:GOOS() - 86ns
[TRACE] 09:37:30.765 terminal.go:Getenv(SSH_CLIENT) - 4.152µs
[TRACE] 09:37:30.765 terminal.go:GOOS() - 48ns
[DEBUG] 09:37:30.765 file.go:Get:80 → getting cache key: environment_platform with value: fedora
[DEBUG] 09:37:30.765 properties.go:GetString:28 → home_icon: ~
[DEBUG] 09:37:30.765 terminal_unix.go:Platform:92 → fedora
[DEBUG] 09:37:30.765 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[DEBUG] 09:37:30.765 home.go:12 → /home/pascal
[TRACE] 09:37:30.765 separator.go:Separator() - 55ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 62ns
[DEBUG] 09:37:30.765 segment.go:Execute:113 → segment: Git
[TRACE] 09:37:30.765 terminal.go:GOOS() - 47ns
[TRACE] 09:37:30.765 separator.go:Separator() - 52ns
[DEBUG] 09:37:30.765 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.765 segment.go:isToggled:234 → no toggles found
[TRACE] 09:37:30.765 terminal.go:GOOS() - 60ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 44ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 48ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 44ns
[TRACE] 09:37:30.765 terminal.go:Flags() - 146ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 36ns
[DEBUG] 09:37:30.765 terminal.go:ResolveSymlink:208 → /mnt/data/dev/ltetools
[TRACE] 09:37:30.765 terminal.go:ResolveSymlink(/mnt/data/dev/ltetools) - 23.679µs
[DEBUG] 09:37:30.765 home.go:12 → /home/pascal
[DEBUG] 09:37:30.765 properties.go:GetString:28 → gitdir_format: 
[TRACE] 09:37:30.765 terminal.go:GOOS() - 89ns
[DEBUG] 09:37:30.765 segment.go:Execute:113 → segment: Root
[DEBUG] 09:37:30.765 properties.go:GetString:28 → style: folder
[DEBUG] 09:37:30.765 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 09:37:30.765 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.765 properties.go:GetString:28 → folder_separator_template: 
[DEBUG] 09:37:30.765 properties.go:GetString:28 → folder_separator_icon: /
[TRACE] 09:37:30.765 terminal_unix.go:Root() - 856ns
[ERROR] 09:37:30.765 terminal.go:HasParentFilePath:477 → stat .git: no such file or directory
[DEBUG] 09:37:30.765 properties.go:GetBool:22 → cycle_folder_separator: false
[TRACE] 09:37:30.765 terminal.go:HasParentFilePath(.git) - 76.063µs
[DEBUG] 09:37:30.765 properties.go:GetString:28 → folder_format: %s
[DEBUG] 09:37:30.765 properties.go:GetString:28 → edge_format: %s
[DEBUG] 09:37:30.765 properties.go:GetString:28 → left_format: %s
[DEBUG] 09:37:30.765 properties.go:GetString:28 → right_format: %s
[TRACE] 09:37:30.765 text.go:Render(ltetools) - 217ns
[TRACE] 09:37:30.765 terminal.go:Flags() - 64ns
[TRACE] 09:37:30.765 terminal.go:GOOS() - 62ns
[TRACE] 09:37:30.765 terminal.go:StackCount() - 93ns
[TRACE] 09:37:30.765 terminal_unix.go:DirIsWritable(/mnt/data/dev/ltetools) - 5.826µs
[DEBUG] 09:37:30.766 terminal.go:RunCommand:293 → NO DATA
[TRACE] 09:37:30.766 terminal.go:RunCommand(who am i) - 1.718799ms
[DEBUG] 09:37:30.766 text.go:patchTemplate:125 →  {{ if .Data.SSHSession }} {{ end }}{{ .UserName }} 
[TRACE] 09:37:30.767 text.go:Render( {{ if .SSHSession }} {{ end }}{{ .UserName }} ) - 542.562µs
[TRACE] 09:37:30.767 text.go:Render(diamond) - 196ns
[DEBUG] 09:37:30.768 text.go:patchTemplate:125 →   {{ path .Data.Path .Data.Location }} 
[TRACE] 09:37:30.768 text.go:Render(  {{ path .Path .Location }} ) - 197.442µs
[TRACE] 09:37:30.768 text.go:Render(powerline) - 110ns
[TRACE] 09:37:30.768 text.go:Render(powerline) - 64ns
[TRACE] 09:37:30.768 text.go:Render(powerline) - 38ns
[DEBUG] 09:37:30.768 text.go:patchTemplate:125 →  {{ if gt .Code 0 }}{{ else }}{{ end }} 
[TRACE] 09:37:30.768 text.go:Render( {{ if gt .Code 0 }}{{ else }}{{ end }} ) - 37.78µs
[DEBUG] 09:37:30.768 text.go:patchTemplate:125 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 09:37:30.768 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 27.705µs
[TRACE] 09:37:30.768 text.go:Render(diamond) - 109ns
[DEBUG] 09:37:30.768 text.go:patchTemplate:125 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 09:37:30.768 text.go:Render({{ if gt .Code 0 }}p:red{{ end }}) - 18.28µs
[TRACE] 09:37:30.768 terminal.go:Shell() - 164ns
[TRACE] 09:37:30.768 terminal.go:Shell() - 51ns
[TRACE] 09:37:30.768 terminal.go:Flags() - 63ns
[TRACE] 09:37:30.768 terminal.go:Flags() - 127ns
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[TRACE] 09:37:30.768 terminal.go:Flags() - 132ns
[TRACE] 09:37:30.768 terminal.go:Flags() - 153ns
[TRACE] 09:37:30.768 terminal.go:GOOS() - 55ns
[TRACE] 09:37:30.768 terminal.go:GOOS() - 83ns
[DEBUG] 09:37:30.768 segment.go:Execute:113 → segment: Node
[DEBUG] 09:37:30.768 segment.go:Execute:113 → segment: Time
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.768 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 09:37:30.768 terminal.go:GOOS() - 82ns
[TRACE] 09:37:30.768 terminal.go:GOOS() - 77ns
[DEBUG] 09:37:30.768 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.768 segment.go:isToggled:234 → no toggles found
[TRACE] 09:37:30.768 terminal.go:Flags() - 142ns
[DEBUG] 09:37:30.768 segment.go:Execute:113 → segment: Python
[DEBUG] 09:37:30.768 properties.go:GetString:28 → time_format: 15:04:05
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 09:37:30.768 terminal.go:GOOS() - 75ns
[DEBUG] 09:37:30.768 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.768 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.768 segment.go:Execute:113 → segment: Shell
[DEBUG] 09:37:30.768 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.768 properties.go:GetString:28 → display_mode: files
[DEBUG] 09:37:30.768 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 09:37:30.768 segment.go:isToggled:234 → no toggles found
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → folders: []
[DEBUG] 09:37:30.768 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 09:37:30.768 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[TRACE] 09:37:30.768 terminal.go:Shell() - 175ns
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → folders: []
[DEBUG] 09:37:30.768 properties.go:GetBool:22 → fetch_package_manager: false
[TRACE] 09:37:30.768 terminal.go:Flags() - 65ns
[DEBUG] 09:37:30.768 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 09:37:30.768 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 09:37:30.768 home.go:12 → /home/pascal
[DEBUG] 09:37:30.768 properties.go:GetString:28 → display_mode: files
[DEBUG] 09:37:30.768 properties.go:GetBool:22 → fetch_virtual_env: false
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(*.js) - 1.167321ms
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(*.py) - 1.261325ms
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(*.go) - 1.266659ms
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(go.mod) - 110.168µs
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(go.sum) - 108.691µs
[DEBUG] 09:37:30.769 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.769 terminal.go:HasFilesInDir(*.ts) - 365.393µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(go.work) - 116.517µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(package.json) - 140.3µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(go.work.sum) - 144.569µs
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(.nvmrc) - 102.404µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(*.ipynb) - 570.083µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 179.358µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(pyproject.toml) - 172.329µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 154.43µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(venv.bak) - 120.009µs
[DEBUG] 09:37:30.770 terminal.go:HasFolder:193 → false
[TRACE] 09:37:30.770 terminal.go:HasFolder(.venv) - 19.586µs
[DEBUG] 09:37:30.770 terminal.go:HasFolder:193 → false
[TRACE] 09:37:30.770 terminal.go:HasFolder(venv) - 8.246µs
[DEBUG] 09:37:30.770 terminal.go:HasFolder:193 → false
[TRACE] 09:37:30.770 terminal.go:HasFolder(virtualenv) - 6.855µs
[DEBUG] 09:37:30.770 terminal.go:HasFolder:193 → false
[TRACE] 09:37:30.770 terminal.go:HasFolder(venv-win) - 6.427µs
[DEBUG] 09:37:30.770 terminal.go:HasFilesInDir:164 → false
[DEBUG] 09:37:30.770 terminal.go:HasFolder:193 → false
[TRACE] 09:37:30.770 terminal.go:HasFilesInDir(.vue) - 94.649µs
[TRACE] 09:37:30.770 terminal.go:HasFolder(pyenv-win) - 7.221µs
[TRACE] 09:37:30.770 text.go:Render(plain) - 321ns
[TRACE] 09:37:30.770 text.go:Render(plain) - 190ns
[TRACE] 09:37:30.770 text.go:Render(plain) - 97ns
[DEBUG] 09:37:30.770 text.go:patchTemplate:125 → in <p:blue><b>{{ .Data.Name }}</b></> 
[TRACE] 09:37:30.771 text.go:Render(in <p:blue><b>{{ .Name }}</b></> ) - 279.618µs
[TRACE] 09:37:30.771 text.go:Render(plain) - 130ns
[DEBUG] 09:37:30.771 text.go:patchTemplate:125 → at <p:blue><b>{{ .Data.CurrentDate | date "15:04:05" }}</b></>
[TRACE] 09:37:30.771 text.go:Render(at <p:blue><b>{{ .CurrentDate | date "15:04:05" }}</b></>) - 83.511µs
[TRACE] 09:37:30.771 text.go:Render(plain) - 116ns
[TRACE] 09:37:30.771 terminal.go:Shell() - 137ns
[TRACE] 09:37:30.771 terminal.go:Shell() - 49ns
[TRACE] 09:37:30.771 terminal.go:Flags() - 48ns
[TRACE] 09:37:30.771 terminal.go:Shell() - 26ns
[DEBUG] 09:37:30.771 terminal_unix.go:TerminalWidth:85 → terminal width: 238
[TRACE] 09:37:30.771 terminal_unix.go:TerminalWidth() - 736.662µs
[DEBUG] 09:37:30.771 home.go:12 → /home/pascal
[TRACE] 09:37:30.771 path.go:Path() - 17.399µs
[TRACE] 09:37:30.772 terminal.go:Flags() - 57ns
```

--- a/src/config/load.go
+++ b/src/config/load.go
@@ -32,15 +32,11 @@ const (
 func Load(configFile, sh string, migrate bool) (*Config, string) {
 	defer log.Trace(time.Now())
 
-	if len(configFile) == 0 {
-		log.Debug("no config file specified, using default")
-		return Default(false), defaultHash
-	}
-
 	configFile, err := filePath(configFile)
 	if err != nil {
 		log.Error(err)
-		return Default(true), defaultHash
+		warning := !errors.Is(err, ErrNoConfig)
+		return Default(warning), defaultHash
 	}
 
 	cfg, hash := readConfig(configFile)
@@ -91,6 +87,9 @@ func Load(configFile, sh string, migrate bool) (*Config, string) {
 	return cfg, hash
 }
 
+// custom no config error
+var ErrNoConfig = errors.New("no config file specified")
+
 func filePath(config string) (string, error) {
 	defer log.Trace(time.Now())
 
@@ -105,7 +104,7 @@ func filePath(config string) (string, error) {
 	}
 
 	if !hasConfig {
-		return "", errors.New("no config file specified")
+		return "", ErrNoConfig
 	}
 
 	if strings.HasPrefix(config, "https://") {

